utils/utils.c

```diff
@@ -92,8 +92,8 @@ finiGbConfig(void)
 int
 glusterBlockSetLogLevel(unsigned int logLevel)
 {
-  char *dom;
-  int level;
+  char *dom = NULL;
+  int level = GB_LOG_CRIT;
```

Comment:
@lxbsz I would rather feel its better to:
```

if (gbCtx == GB_CLI_MODE) {
    dom = "cli";
    level = GB_LOG_DEBUG;
  } else {
    dom = "mgmt";
    level = GB_LOG_CRIT;
  }
```

Reply:
@pkalever 
Sure, looks good to me.
Will fix it.
Thanks.